app.js

```diff
@@ -170,7 +170,7 @@ app
 
 if (!module.parent) {
     models.sequelize.sync().then(() => {
-        app.listen(process.env.NEAR_FIDDLE_PORT || 3000);
+        app.listen(3000);
```

Comment:
let's take in `PORT` variable, it's useful when you run it locally

Reply:
I thought `PORT` variable is node-specific and it would work even if we do not explicitly allow it, correct?

Reply:
No

Reply:
It's just a general purpose language basically, you can like go and start 10 different servers on different ports within single script.